config/rbac/role.yaml

```diff
@@ -154,6 +154,12 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
```

Comment:
Hey @JohnStrunk 

Probably intended but I wonder if the operator should specify only the scribe scc names with 
```yaml
  resourceNames:
    - scribe-rsync
    - scribe-mover
```

Like this -
https://github.com/backube/scribe/blob/8fb32d24b73edaf6a7bfdbc5eb565c2623aad975/mover-rsync/deploy-destination.yaml#L58-L65

Reply:
Yeah... let me see if I can add that